docs/UserGuide.md

```diff
@@ -239,7 +239,7 @@ Examples:
   5. has strictly more than 3 days of overtime
 
 * (To be updated) `find n/alex david l/<3` returns `Alex Yeoh`, `David Li` as long as they have less than 3 leaves left.<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
+  ![result for 'find alex david'](images/user-guide/findAlexDavidResult.png)
```

Comment:
Maybe this screenshot could be updated based on the latest UG example? I think it is a search on Carl and Elle.

Reply:
I was thinking of using Alex and David because it is the default dataset, and may be confusing to users since Carl and Elle are from imported data. But I have also added the result for Carl and Elle in as well.